Tools/Scripts/build-and-analyze

```diff
@@ -139,12 +139,19 @@ def main(args):
     make_command = ['make', 'analyze', 'SDKROOT={}'.format(args.sdkroot)]
     if args.analyzer_path:
         make_command.extend(['CC={}'.format(analyzer_path), 'CPLUSPLUS={}'.format(analyzer_path)])
+    if args.preprocessor_additions:
+        make_command.extend(['GCC_PREPROCESSOR_ADDITIONS={}'.format(args.preprocessor_additions)])
```

Comment:
Do we need double-quotes around `'GCC_PREPROCESSOR_ADDITIONS="{}"' in case there are more than one?

Reply:
So I tried that but that results in the second macro be interpreted as the value of the first macro.
e.g. `A=1 B=1` would be interpreted as `#define A 1 B=1` instead of `#define A 1` and `#define B 1`.